x/programs/cmd/simulator/src/util.rs

```diff
@@ -0,0 +1,16 @@
+use std::path::Path;
+pub(crate) fn get_path() -> &'static str {
+    let path = env!("SIMULATOR_PATH");
+
+    if !Path::new(path).exists() {
+        eprintln!();
+        eprintln!("Simulator binary not found at path: {path}");
+        eprintln!();
+        eprintln!("Please run `cargo clean -p simulator` and rebuild your dependent crate.");
+        eprintln!();
```

Comment:
```suggestion
        eprint!(r#"
Simulator binary not found at path: {path}

Please run `cargo clean -p simulator` and rebuild your dependent crate.

"#);
```
Or whatever the equivalent is